### PR TITLE
#[139551325] Fixing front tests 

### PR DESCRIPTION
**Pull request by NgangaMaryanne**
### What does this PR do?
It updates the front end tests, adding four tests

### Description of Task to be completed?
**_hc/front/tests/test_add_channel.py_**
Test that the team access works. 
Test that bad channel kinds don't work.

**_hc/front/tests/test_add_check.py_**
Test that team access works.

**_hc/front/tests/test_add_pushbover.py_**
Test that pushover validates priority.

### How should this be manually tested?
Clone repo
Setup virtual environment and install requirements.
   run **`./manage.py test`**

### Any background context you want to provide?
Following the instructions on the healthchecks tests.

### What are the relevant pivotal tracker stories?
 \#139551325

### CODE REVIEW

File: hc/front/tests/test_add_pushbover.py [hunk 1]
```diff
@@ -38,3 +38,19 @@ def test_it_validates_nonce(self):
...
+        self.client.login(username="alice@example.org", password="password")
+
+        session = self.client.session
+        session["po_nonce"] = "n"
+        session.save()
+
+        params = "pushover_user_key=a&nonce=n"
+        r = self.client.get("/integrations/add_pushover/?%s" % params)
+        self.assertEqual(r.status_code, 400)
+
```
Comment (osule) [inline hc/front/tests/test_add_pushbover.py]: There are so many blank lines 🙈 
Let's do a PEP8 lint.

File: hc/front/tests/test_add_check.py [hunk 2]
```diff
@@ -1,5 +1,8 @@
 from hc.api.models import Check
+
 from hc.test import BaseTestCase
+from hc.api.models import Channel
```
Comment (osule) [inline hc/front/tests/test_add_check.py]: It's easier to track imports from the same module if we collect such imports together. We should import Channel together with Check.

File: hc/front/tests/test_add_channel.py [hunk 3]
```diff
@@ -38,4 +38,23 @@ def test_instructions_work(self):
             self.assertContains(r, "Integration Settings", status_code=200)
 
     ### Test that the team access works
+    def test_team_access_works(self):
+        url = "/integrations/add/"
+        form = {"kind": "slack", "value": {"team_name": "Andela"}}
+        #using bob
+        self.client.login(username="bob@example.org", password="password")
```
Comment (osule) [inline hc/front/tests/test_add_channel.py:45]: Ah, nice!

File: hc/front/tests/test_add_pushbover.py (hunk 1 above)
Comment (NgangaMaryanne) [reply hc/front/tests/test_add_pushbover.py]: Formatted with auto pep8

File: hc/front/tests/test_add_check.py (hunk 2 above)
Comment (NgangaMaryanne) [reply hc/front/tests/test_add_check.py]: Resolved.

**Review by osule — COMMENTED** [review]
LGTM

**Pull request merged**